src/Chirp.Infrastructure/Repositories/CheepRepository.cs

```diff
@@ -61,5 +75,18 @@ public List<CheepDTO> GetCheepsFromAuthor(int page, string author)
 				})
 				.ToList();
 		}
+		private List<CheepDTO> GetCheepsFromAuthor(string author)
+		{
+			return _context.Cheeps
+				.Where(c => c.Author.Name == author)
+				.OrderByDescending(c => c.TimeStamp)
+				.Select(c => new CheepDTO
+				{
+					AuthorName = c.Author.Name,
+					Message = c.Text,
+					TimeStamp = c.TimeStamp.ToString()
```

Comment:
c.TimeStamp.ToString() should probably take the formatting argument like the other GetCheepsFromAuthor method. ("yyyy-MM-dd H:mm:ss")

Reply:
This has been implemented! Thank you for noticing!